tests/run-make/rustdoc-error-lines/rmake.rs

```diff
@@ -5,7 +5,6 @@ use run_make_support::rustdoc;
 
 fn main() {
     let output = rustdoc().input("input.rs").arg("--test").run_fail().stdout_utf8();
-
```

Comment:
Question: where did these formatting diffs from come?

Reply:
Just me waving around autoformatting a little too liberally, I have restored it.